cli/pkg/local/server.go

```diff
@@ -450,12 +440,13 @@ func (s *Server) PushToGithub(ctx context.Context, r *connect.Request[localv1.Pu
 	}), nil
 }
 
+// DeployProject implements localv1connect.LocalServiceHandler.
 func (s *Server) DeployProject(ctx context.Context, r *connect.Request[localv1.DeployProjectRequest]) (*connect.Response[localv1.DeployProjectResponse], error) {
+	// Get authenticated admin client
 	if !s.app.ch.IsAuthenticated() {
-		return nil, errors.New("user should be authenticated before deploying")
+		return nil, errors.New("must authenticate before performing this action")
 	}
-	// Get admin client
-	c, err := client.New(s.app.adminURL, s.app.ch.AdminTokenDefault, "Rill Localhost")
+	c, err := s.app.ch.Client()
```

Comment:
Note that this enables the `cmdutil.Helper` to manage a single connection to the `admin` service and correctly reload it if the underlying config changes.